--- TGeant3/TGeant3.h_old	2018-07-27 16:48:54.127400497 +0200
+++ TGeant3/TGeant3.h	2018-07-27 16:58:14.701105254 +0200
@@ -676,9 +676,16 @@
   Double_t Xsec(char* reac, Double_t energy, Int_t part, Int_t mate);
   void  TrackPosition(TLorentzVector &xyz) const;
   void  TrackPosition(Double_t &x, Double_t &y, Double_t &z) const;
+  void  TrackPosition(Float_t &x, Float_t &y, Float_t &z ) const {
+    TrackPosition( (Double_t&)x, (Double_t&)y, (Double_t&)z );
+  }
   void  TrackMomentum(TLorentzVector &xyz) const;
   void  TrackMomentum(Double_t &px, Double_t &py, Double_t &pz,
                       Double_t &etot) const;
+  void  TrackMomentum(Float_t &px, Float_t &py, Float_t &pz,
+                      Float_t &etot) const {
+    TrackMomentum( (Double_t&)px, (Double_t&)py, (Double_t&)pz, (Double_t&)etot );
+  }
   Int_t NofVolumes() const;
   Int_t NofVolDaughters(const char* volName) const;
   const char*  VolDaughterName(const char* volName, Int_t i) const;
